js/controller.js

```diff
@@ -61,9 +61,23 @@
                     console.log(error);
                 });
 
+                setTimeout(function() { refreshFitbitData(); }, 1000);
```

Comment:
This function appears a little misleading at first - its actually just a 1 second delay before the fitbit data is refreshed once every "MirrorRefresh" cycle. This means this function is only actually being called once every mirror refresh cycle which I believe by default is every 60 minutes using the $interval timer with Angular. The reason I delay it by 1 second, is to give the fitbit-service controller time to fully initialize when the Smart Mirror app starts up for the first time and fitbit integration is enabled in config. Otherwise I noticed the initial API call can fail as the controller hasn't finished initializing. 

So now the ways fitbit data refreshes are:
- Mirror Refresh (once an hour by default)
- Voice command

and they only actually call any refresh code if the user has enabled fitbit in their config.js :smile: